crates/sui-indexer/src/store/pg_partition_manager.rs

```diff
@@ -44,16 +46,24 @@ GROUP BY table_name;
 
 pub struct PgPartitionManager<T: R2D2Connection + 'static> {
     cp: ConnectionPool<T>,
+    partition_strategies: HashMap<&'static str, PgPartitionStrategy>,
 }
 
 impl<T: R2D2Connection> Clone for PgPartitionManager<T> {
```

Comment:
Out of interest -- why can't this be `derived`?

Reply:
just tried deriving `Clone` on `PgPartitionManager`, R2D2Connection trait doesn't derive or impl clone so we'd need to update all the places where we have `T: R2D2Connection + 'static`